src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorSpectrum.cs

```diff
@@ -1066,7 +1086,7 @@ private async void CreateBitmapsAndColorMap()
 			newHsvValues.Capacity = pixelCount;
 
 			int minDimensionInt = (int)Math.Round(minDimension);
-			//WorkItemHandler workItemHandler = 
```

Comment:
This was a last minute change to just use standard C# async/await pattern with Task instead of whatever is done in WinRT. The only downside was loss of cancellation support.

Of course I forgot to add an Uno Doc for it. Consider this a reminder to myself to document this later.